packages/common/src/interfaces.ts

```diff
@@ -72,6 +72,7 @@ export interface StateManagerInterface {
   modifyAccountFields(address: Address, accountFields: AccountFields): Promise<void>
   putContractCode(address: Address, value: Uint8Array): Promise<void>
   getContractCode(address: Address): Promise<Uint8Array>
+  getContractCodeSize(address: Address): Promise<number>
```

Comment:
This is a breaking change, we need to make this optional.

Reply:
done :pray: